src/Types/BlockHeight.cs

```diff
@@ -40,6 +46,9 @@ public BlocksAtHeightRequest Into() =>
 /// </param>
 public sealed record RelativeHeight(ulong Height, uint GenesisIndex, bool Restrict) : IBlockHeight
 {
+    /// <summary>
+    /// Convert an RelativeHeight into a request for blocks at height.
```

Comment:
```suggestion
    /// Convert a RelativeHeight into a request for blocks at height.
```

Also request for blocks at what height? I guess I'm missing some context but the sentence just sounds weird. Same in the other places.

Reply:
I change the doc comment to make it more clear